test/memote.jl

```diff
@@ -70,3 +70,22 @@ end
 
     @test length(reactions_transport_no_gpr(model; config = memote_config)) == 4
 end
+
+@testset "Biomass" begin
+    @test model_has_atpm_reaction(model)
+    wrong_model = convert(StandardModel, model)
+    remove_reaction!(wrong_model, "ATPM")
+    @test !model_has_atpm_reaction(wrong_model)
+
+    @test all(model_has_growth_atp_in_biomass(model))
+
+    @test "BIOMASS_Ecoli_core_w_GAM" in model_biomass_reactions(model)
+
+    @test first(model_biomass_is_consistent(model)) == 1.5407660614638816
+
+    @test model_solves_default(model, Tulip.Optimizer)
+
+    @test length(model_can_produce_biomass_precursors(model, Tulip.Optimizer)) == 3
+
+    @test length(first(model_biomass_reactions(model))) == 24
```

Comment:
this is the length of the string ID? that's already tested above.

Reply:
whoops, tested the wrong function